madrasthemes/mas-elementor
# Feature/container swiper arrows

### Steps to test 

1. Added cart button active style controls in add to cart widget

- Add cart icon in mas-post and check for Icon tab in Style Tab of Add to cart widget
![image](https://github.com/user-attachments/assets/1e8c77d2-1fdf-46bf-af47-2f04e4050eba)


2. Added row gap option for product widget
![image](https://github.com/user-attachments/assets/7689f891-9bc9-4dd2-85d5-a2c9bb2f8b79)


3. Elementor 3.24.0 Compatibility - Check by updating the elementor to the latest version and use elementor page
